packages/gatsby/src/commands/develop.ts

```diff
@@ -461,7 +471,15 @@ module.exports = async (program: any) => {
     }
   }
 
-  function printInstructions(appName, urls, useYarn) {
+  function printInstructions(
+    appName: string,
+    urls: {
```

Comment:
Do we want to inline this or create an interface/type for it?

Reply:
(IMO) I think inlining is fine. Allows type definitions to be more terse and also allow us to avoid naming things which is sometimes just hard and not strictly necessary.

I would actually default to inlining unless it cause more problems when you need to use one of those nested types?

Reply:
i can type it

Reply:
all good for me 👍 I like inlining for functions too unless I need to use it in 2 places.